core-common/src/main/java/org/zalando/nakadi/domain/NakadiMetadata.java

```diff
@@ -0,0 +1,11 @@
+package org.zalando.nakadi.domain;
+
+public interface NakadiMetadata {
+
+    String getEid();
+
+    String getPartition();
+
+    String getEventType();
```

Comment:
Adding `metadataVersion` as well can be useful. 

```suggestion
    String getEventType();
    
    byte getMetadataVersion();
```
This way a metadata can also tell which version it is.